web/apps/labelstudio/src/pages/Settings/AnnotationSettings.jsx

```diff
@@ -36,16 +36,13 @@ export const AnnotationSettings = () => {
           >
             <Form.Row columnCount={1}>
               <Elem name={"header"}>Labeling Instructions</Elem>
+              <div style={{ color: "var(--sand_600)", fontSize: ".875rem" }}>
```

Comment:
```suggestion
              <div style={{ color: "var(--sand_600)", fontSize: "14px" }}>
```

Reply:
Given all the surrounding values are px-based, though I think REM is the most correct unit here, local consistency is most important.